docs/MAINTAINING.md

```diff
@@ -62,4 +65,14 @@ It also gives us a way to separate the issue triage from bigger-picture, long-te
 
 `<major.minor.patch>`
 
-The code-server project follows traditional [semantic versioning](ttps://semver.org/), with the objective of minimizing major changes that break backward compatibility. We increment the patch level for all releases, except when the upstream Visual Studio Code project increments its minor version or we change the plugin API in a backward-compatible manner. In those cases, we increment the minor version rather than the patch level.
+The code-server project follows traditional [semantic versioning](https://semver.org/), with the objective of minimizing major changes that break backward compatibility. We increment the patch level for all releases, except when the upstream Visual Studio Code project increments its minor version or we change the plugin API in a backward-compatible manner. In those cases, we increment the minor version rather than the patch level.
+
+## Pull Requests
+
+Ideally, every PR should fix an issue. If it doesn't, make sure it's associated with a version milestone.
+
+If a PR does fix an issue, don't add it to the version milestone. Otherwise, the version milestone will have duplicate information: the issue & the PR fixing the issue.
+
+### Merge Strategies
+
+For most things, we recommend "Squash and Merge". If you're updating `lib/vscode`, we suggest using the "Rebase and Merge" strategy. There may be times where "Create a merge commit" makes sense as well. Use your best judgement.
```

Comment:
Awesome to see this included.

Reply:
Thanks for fixing that typo too!

Reply:
Examples are always nice - for the "Create a merge commit" flow, if you have any. You might also want to mention that if you're unsure, to discuss with the team?

Reply:
I don't have a good example off the top of my head but if you have one and want to add it, feel free to add in a follow-up PR? I'll add that last part about discussing with the team though! Good idea.